frontend/src/customHooks/useUser.ts

```diff
@@ -46,9 +47,22 @@ export default function useUser(loadAllGames: () => void) {
                 setUser(undefined);
             })
             .catch(() => {
-                toast.error("Logout hat nicht funktioniert:");
+                console.log("error")
             });
     }
 
-    return { user, login, logout, isLoading };
+    const createUser = async (newUser: UserModel) => {
+        return await axios.post("/api/users/signup", newUser, {
+            withCredentials: true
+        }).then((response) => {
+            setUser(response.data)
+            return true;
+        }).catch((error) => {
+            console.error(error);
+            return false;
+        })
+    }
```

Comment:
I'm not sure for what you use the true or false. I think you can use a callback method instead. 
Just like you give a method to props you can give a method to the parameters: 
`(newUser: UserModel, successCallback: () => void`
and call it with: `() => successCallback()`

when I guess the usage of the return true / false right you could remove the await as a plus.

Reply:
yeah i don't know how to usw the Callback to be honest